Assets/MRTK/Core/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs

```diff
@@ -15,6 +15,18 @@ namespace Microsoft.MixedReality.Toolkit.Input.Editor
     [CustomEditor(typeof(MixedRealityControllerMappingProfile))]
     public class MixedRealityControllerMappingProfileInspector : BaseMixedRealityToolkitConfigurationProfileInspector
     {
+        private readonly struct ControllerMappingSignature
+        {
+            public SupportedControllerType SupportedControllerType;
+            public Handedness Handedness;
```

Comment:
The build issue is due to non-readonly fields / properties in a readonly struct. Converting these into auto-properties with public getters should fix it:

```suggestion
            public SupportedControllerType SupportedControllerType { get; }
            public Handedness Handedness { get; }
```